test/end-to-end/pages/AuthPage.js

```diff
@@ -0,0 +1,30 @@
+import { ST_ROOT_SELECTOR, TEST_CLIENT_BASE_URL } from "../../constants";
+
+export class AuthPage {
+    static async navigate(page, testContext = {}) {
+        const testParams = new URLSearchParams(testContext);
+
+        const url = `${TEST_CLIENT_BASE_URL}/auth?${testParams}`;
+        console.log(url);
+
+        await Promise.all([page.goto(url), page.waitForNavigation({ waitUntil: "networkidle0" })]);
+
+        return new AuthPage(page);
+    }
+
+    constructor(page) {
+        this.page = page;
+    }
+
+    async isFeatureMounted() {
```

Comment:
Have you tested that this function returns false if the feature is not there? One way to test this is to intentionally introduce a bug a feature component where it doesn't render the theme, and then see if this returns false.

Reply:
It relies on application being rendered on a route - it will return `false` when the supertokens UI is not mounted.

I can improve the selector, it will be different for each route, but will become more independent on how the top-level components are rendered.

Reply:
I'm going to extract that to an issue, as at this moment it is sufficient to check if the UI was mounted. Tracked in https://github.com/supertokens/supertokens-auth-react/issues/266